### ISSUE TITLE
Error executing mkvmerge

### ISSUE THREAD

**Issue by knbobick** [mbunkus/mkvtoolnix#1853]
I have read https://github.com/mbunkus/mkvtoolnix/wiki/JSON-output-could-not-be-parsed and added both the mkvmerge.exe file and, when that didn't work, the entire folder to the exception list. I still can't add an .m4v and its associated .wav into mkvtoolnix v9.7.1. I am on Win10 64 bit with Symantec Endpoint 12.1.6. As a just in case, I turned on defender and added an exception there as well and then turned it back off. Same result.
![error](https://cloud.githubusercontent.com/assets/22716205/21912055/ab0cab58-d8f1-11e6-91ba-c19da8b2cb3e.jpg)
![exceptions](https://cloud.githubusercontent.com/assets/22716205/21912054/ab0b86a6-d8f1-11e6-8408-8d4b432acf53.jpg)

**Comment by mbunkus** [comment]
Thanks for the report. Can you please upload the content of the [cache folder](/mbunkus/mkvtoolnix/wiki/Location-of-settings,-job-queue,-cache-folder) as well as one of the files for which the error occurs to my [FTP server](/mbunkus/mkvtoolnix/wiki/FTP-server) so that I can take a look at it? Thanks.

**Comment by knbobick** [comment]
As there were several files in the cache, I cleared the cache and tried to import the offending file again and it worked this time. Curious, I restored the files back to the cache and tried again and it no longer worked. Do you still want the files or do want to claim victory and update the other post to tell folks to also clear the cache?

**Comment by mbunkus** [comment]
Yes, I definitely want the cache files. Several other folks have reported similar issues, and so far I haven't been able to track down the root cause as I haven't been able to reproduce such problems on my machines yet. Clearing the cache is a workaround, but I'd like to fix the underlying bug so that clearing isn't necessary. Having the cache files would definitely help.

Only upload the cache folder, not the files for which identification failed before. As the files are working now I don't think it is connected to the actual files — it's more likely a subtle race condition or I/O not being read fully.

Thanks.

**Comment by knbobick** [comment]
I just uploaded a zip of all of the files in the cache. Let me know if there is anything else I can help with.

**Comment by mbunkus** [comment]
Thanks a lot. Is there anything else you can tell me about the files which failed to identify? For example:

* Where did you add them from (internal hard drive, SSD, extern USB drive, network share…)?
* Did you add them one by one or as part of a larger set of files?
* Did you use the "add file" button, the right-click context menu with "add files" or did you drag & drop the file from an external application such as Windows Explorer?

Basically… any kind of information you can think of that may help me in reproducing the problem on my side.

Just to be extra thorough: can you please also upload the `mkvtoolnix-gui.ini` configuration file to my server? See the same wiki article about file locations where to find them.

**Comment by mbunkus** [comment]
Oh! Looking at the files I can see the following output:

```
{
   "cacheMetaData" : {
      "fileName" : "E:\\Home Movies\\1992.m4v",
      "programVersion" : "9.7.1",
      "uiLocale" : "en",
      "fileSize" : 3960761418,
      "fileModificationTime" : 1484212009541
   },
   "data" : {
      "identifier" : {
         "errorTitle" : "Error executing mkvmerge",
         "errorText" : "The JSON output generated by mkvmerge could not be parsed (parser's error message: parse error - unexpected 'm').",
         "output" : [
            "mkvmerge v9.4.2 ('So High') 64bit",
…
```

This means that you ran the GUI (`mkvtoolnix-gui.exe`) in version 9.7.1 together with mkvmerge (`mkvmerge.exe`) v9.4.2. That won't work. The GUI and mkvmerge must always be the same version.

I'm close this issue as it's definitely not a bug in MKVToolNix. I'll also add a note to the wiki page about such a possibility.
